includes/feedzy-rss-feeds-feed-tweaks.php

```diff
@@ -92,6 +92,66 @@ function display_external_post_image( $html, $post_id, $post_thumbnail_id, $size
 }
 add_filter( 'post_thumbnail_html', 'display_external_post_image', 10, 5 );
 
+/**
+ * Filters the Attachment Image HTML
+ *
+ * @param string $html the attachment image HTML string.
+ * @param array  $settings       Control settings.
+ * @param string $image_size_key Settings key for image size.
+ * @param string $image_key      Settings key for image.
+ */
+function elementor_display_external_post_image( $html, $settings, $image_size_key, $image_key ) {
```

Comment:
@girishpanchal30, we should use feedzy based prefix, not elementor, to avoid scenarios when they might have a similar one in their code :) 

Thank you!

Reply:
@selul Done!